src/markdown/tutorial/part-1/05-more-about-components.md

```diff
@@ -131,119 +137,129 @@ wait  .rentals li:nth-of-type(3) article.rental
 
 Things are looking pretty convincing already; not bad for just a little bit of work!
 
-## Organizing Code with Namespaced Components
+## Organizing Code in Folders
 
 Next, let's add the image for the rental property. We will use the component generator for this again:
 
 ```run:command cwd=super-rentals
 ember generate component rental/image
 ```
 
-This time, we had a `/` in the component's name. This resulted in the component being created at `app/components/rental/image.hbs`, which can be invoked as `<Rental::Image>`.
+This time, we had a `/` in the component's name. This resulted in the component being created at `app/components/rental/image.gjs`.
 
 ```run:command hidden=true cwd=super-rentals
 ember test --path dist
-git add app/components/rental/image.hbs
-git add tests/integration/components/rental/image-test.js
+git add app/components/rental/image.gjs
+git add tests/integration/components/rental/image-test.gjs
 ```
 
-Components like these are known as *[namespaced](https://en.wikipedia.org/wiki/Namespace)* components. Namespacing allows us to organize our components by folders according to their purpose. This is completely optional&mdash;namespaced components are not special in any way.
+We can organize our components by folders according to their purpose. This is completely optional&mdash;these components are not special in any way.
 
 ## Forwarding HTML Attributes with `...attributes`
 
 Let's edit the component's template:
 
-```run:file:patch lang=handlebars cwd=super-rentals filename=app/components/rental/image.hbs
-@@ -1,1 +1,3 @@
--{{yield}}
-\ No newline at end of file
-+<div class="image">
-+  <img ...attributes>
-+</div>
+```run:file:patch lang=gjs cwd=super-rentals filename=app/components/rental/image.gjs
+@@ -1,3 +1,5 @@
+ <template>
+-  {{yield}}
++  <div class="image">
++    <img ...attributes />
++  </div>
+ </template>
 ```
 
 Instead of hard-coding specific values for the `src` and `alt` attributes on the `<img>` tag, we opted for the `...attributes` keyword instead, which is also sometimes referred to as the *["splattributes"](../../../components/component-arguments-and-html-attributes/#toc_html-attributes)* syntax. This allows arbitrary HTML attributes to be passed in when invoking this component, like so:
 
-```run:file:patch lang=handlebars cwd=super-rentals filename=app/components/rental.hbs
-@@ -1,2 +1,6 @@
- <article class="rental">
-+  <Rental::Image
-+    src="https://upload.wikimedia.org/wikipedia/commons/c/cb/Crane_estate_(5).jpg"
-+    alt="A picture of Grand Old Mansion"
-+  />
-   <div class="details">
+```run:file:patch lang=gjs cwd=super-rentals filename=app/components/rental.gjs
+@@ -1,3 +1,9 @@
++import RentalImage from 'super-rentals/components/rental/image';
```

Comment:
this should be a relative import. there is no sense in using all this path when the files are siblings.

```suggestion
+import { RentalImage } from './rental/image';
```
(I also changed to named export, so that names have meaning again -- default export was only required because the framework called these components. Now we are, so we can use consistent naming via named exports)